api/db-cdr/changelog-schema/db.changelog-20.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't need this file. We can just remove the add index part from this folder.